README.md

```diff
@@ -83,30 +83,50 @@ You can click on the printed URLs to open them in a browser.
 
 ### Using your local machine
 
-⚠️ these instructions are out of date.
-
-Install node.js v16:
-
-- Install with [nodenv](https://github.com/nodenv/nodenv)
-- or [nvm](https://github.com/nvm-sh/nvm)
-
-Run the setup script:
-
-```
-$ script/dev_setup
-```
-
-Run the dev server:
-
-```
-$ yarn dev
-```
-
-And then view graphql sandbox at http://localhost:3000/graphql
-
-If you run into problems setting up your development environment please create an issue describing any errors you encounter.
-
-See [the README in the `frontend` directory](/frontend/README.md) for instructions on setting up for front end development.
+1. Install Node.js 16:
+   1. Install with [nodenv](https://github.com/nodenv/nodenv)
+   2. or [nvm](https://github.com/nvm-sh/nvm)
+1. Switch to Node 16 if needed. nodenv should do it automatically, but with nvm you'll need to run `nvm use 16`
+1. Install the dependencies: `yarn install --frozen-lockfile`
+1. Build the project: `yarn build`
+1. Install postgresql: `brew install postgresql@13`
```

Comment:
**nitpick**: `brew` is OS X only. So Linux and Windows users will fail with these instructions.